eoscostarica/guias.eoscostarica.io
# feat(frontend): Improved footer design ESP #471

### Improve footer styles

### What does this PR do?

- resolves https://github.com/eoscostarica/guide.eoscostarica.io/issues/471

Note: react-icons was installed: npm install react-icons --save or yarn install react-icons

### Steps to test
1. Navigate to the bottom of [guias.eosio](https://guias.eoscostarica.io/) and see page footer

#### CheckList
